ext/openssl/tests/openssl_pkcs7_sign_basic.phpt

```diff
@@ -47,11 +47,7 @@ NULL
 
 Warning: openssl_pkcs7_sign(): error opening input file %s in %s on line %d
 bool(false)
-
-Warning: openssl_pkcs7_sign(): error opening input file %s in %s on line %d
```

Comment:
Why do we get less errors here?

Reply:
This is about empty filenames (`""`); `expand_filepath_with_mode()` bails out without error message in this case:
https://github.com/php/php-src/blob/6871a49b6609a98e291248b0e89ae9730d9dd580/main/fopen_wrappers.c#L762-L764
Not sure whether we should do something about that.

Reply:
It looks like it does other path validity checks as part of https://github.com/php/php-src/blob/6871a49b6609a98e291248b0e89ae9730d9dd580/main/fopen_wrappers.c#L811-L814
So if that's the case I think there should be some kind of error kept in openssl.c as otherwise it's basically suppressing errors for situations where we had error before because `BIO_new_file` would fail...

Reply:
I agree that this makes sense, although it is not done everywhere (e.g. `var_dump(disk_total_space("") does not warn)`, but that is another issue.